strapi/strapi
# [DEITS] Local Strapi Source Provider - Links Streaming

### What does it do?

- Update the Local Strapi Source Provider to include links streaming capabilities
- Add tests for the provider

### Why is it needed?

- We need to be able to stream the links from a local strapi source

### How to test it?

- Create a transfer engine instance with
  - A Local Strapi Source Provider
  - Any destination provider
- Transfer the data
- See that you're receiving links as expected


### Todo
- [x] Need to update typings to remove any
- [x] Add missing test for the links.ts utils
